test_scripts/RC/InteriorVehicleData_cache/common_interiorVDcache.lua

```diff
@@ -0,0 +1,349 @@
+    hmiResponseParams = function(pModuleType, pSubscribe)
+      return {
+        moduleData = m.getModuleControlData(pModuleType, m.actualInteriorDataStateOnHMI[pModuleType]),
+        isSubscribed = pSubscribe
+      }
+    end,
+    responseParams = function(success, resultCode, pModuleType, pSubscribe)
+      return {
+        success = success,
+        resultCode = resultCode,
+        moduleData = m.getModuleControlData(pModuleType, m.actualInteriorDataStateOnHMI[pModuleType]),
+        isSubscribed = pSubscribe
+      }
+    end
+  },
+  OnInteriorVehicleData = {
+    appEventName = "OnInteriorVehicleData",
+    hmiEventName = "RC.OnInteriorVehicleData",
+    hmiResponseParams = function(pModuleType, pParams)
+      return {
+        moduleData = m.getModuleControlData(pModuleType, pParams)
+      }
+    end,
+    responseParams = function(pModuleType, pParams)
+      return {
+        moduleData = m.getModuleControlData(pModuleType, pParams)
+      }
+    end
+  }
+}
+
+function m.getHMIRequestParams(pRPC, ...)
+  return rcRPCs[pRPC].hmiRequestParams(...)
+end
+
+function m.getHMIResponseParams(pRPC, ...)
+  return rcRPCs[pRPC].hmiResponseParams(...)
+end
+
+function m.getResponseParams(pRPC, ...)
+  return rcRPCs[pRPC].responseParams(...)
+end
+
+function m.getAnotherModuleControlData(module_type)
+  local out = {}
+  if module_type == "CLIMATE" then
+    out = {
+      fanSpeed = 65,
+      currentTemperature = {
+        unit = "FAHRENHEIT",
+        value = 44.3
+      },
+      desiredTemperature = {
+        unit = "CELSIUS",
+        value = 22.6
+      },
+      acEnable = false,
+      circulateAirEnable = false,
+      autoModeEnable = true,
+      defrostZone = "ALL",
+      dualModeEnable = true,
+      acMaxEnable = false,
+      ventilationMode = "UPPER"
+    }
+  elseif module_type == "RADIO" then
+    out = {
+      frequencyInteger = 1,
+      frequencyFraction = 2,
+      band = "AM",
+      rdsData = {
+        PS = "ps",
+        RT = "rt",
+        CT = "123456789012345678901234",
+        PI = "pi",
+        PTY = 2,
+        TP = false,
+        TA = true,
+        REG = "US"
+      },
+      availableHDs = 1,
+      hdChannel = 1,
+      signalStrength = 5,
+      signalChangeThreshold = 20,
+      radioEnable = true,
+      state = "ACQUIRING"
+    }
+  end
+  return out
+end
+
+function m.unregistrationApp(pAppId, isHMIreqExpect, pModuleType)
+  local rpc = "GetInteriorVehicleData"
+  local HMIrequestsNumber
+  if isHMIreqExpect == nil then isHMIreqExpect = false end
+  if isHMIreqExpect == true then
+  HMIrequestsNumber = 1
+  else
+  HMIrequestsNumber = 0
+  end
+  EXPECT_HMICALL(commonRC.getHMIEventName(rpc), m.getHMIRequestParams(rpc, pModuleType, pAppId, false))
+  :Do(function(_, data)
+      m.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS",
+      m.getHMIResponseParams(rpc, pModuleType, false))
+    end)
+  :Times(HMIrequestsNumber)
+  local mobSession = m.getMobileSession(pAppId)
```

Comment:
@GetmanetsIrina `m.getMobileSession(pAppId)` can be used directly without defining of `mobSession`.
The same is for `hmiAppId`